src/Options/Applicative/NonEmpty.hs

```diff
@@ -0,0 +1,17 @@
+{-# LANGUAGE CPP, Rank2Types, ExistentialQuantification #-}
+module Options.Applicative.NonEmpty (
+  some1
+) where
+
+import Data.List.NonEmpty (NonEmpty (..))
+
+import Options.Applicative.Types
+import Control.Applicative
+import Prelude
+
+-- | Sequences an action one or more times.
+--
+--   Functionally identical to 'Data.List.NonEmpty.some1',
+--   but is preferred as it gives a nicer help text.
```

Comment:
This haddockumentation should have a `@since` attribute.
https://haskell-haddock.readthedocs.io/en/latest/markup.html#since